app/assets/stylesheets/show.scss

```diff
@@ -1,6 +1,9 @@
 .show-container {
+  border-radius: 5px;
```

Comment:
Properties should be ordered background-color, border-radius, margin, padding, position